buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+   `kotlin-dsl`
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.21")
+}
+
+//val buildLogicJvmTarget = "11"
```

Comment:
Ditto.

Reply:
Good spot, most of the comments here can be removed. I'll make another PR.